src/ctim/schemas/vocabularies.cljc

```diff
@@ -258,7 +258,8 @@
     "mac_address"
     "file_name"
     "file_path"
-    "odns_identity"})
+    "odns_identity"
+    "odns_identity_label"})
 
 (def-enum-type ObservableTypeIdentifier
   observable-type-identifier
```

Comment:
I think that if we're adding many new specific observable type for different product/context we should make the "ObservableTypeIdentifier" open to any string.